TCPEchoServer/Client.cpp

```diff
@@ -0,0 +1,51 @@
+﻿#include "Socket.h"
+#include <iostream>
+
+int main(int argc, char* argv[]) {
+
+	WSADATA wsaData;
+
+	std::cout << "Input your name: ";
+	std::string s;
+	std::cin >> s;
+
+	int iResult = WSAStartup(MAKEWORD(2, 2), &wsaData);
+	if (iResult != 0)
+	{
+		std::cout << "WSAStartup failed: " << iResult << std::endl;
```

Comment:
when WSAStartup fails it doesn't make sense to proceed. Please return from main() with a meaningless retcode.

Reply:
Fixed

Reply:
Fixed